package.json

```diff
@@ -15,7 +15,7 @@
     "build": "tsc && vite build",
     "preview": "vite preview",
     "check-dist": "if [ ! -d dist ]; then echo '\\e[31mError: The \"dist\" folder does not exist. Please run \"yarn build\" first.\\e[0m'; exit 1; else echo 'Dist folder exists'; fi",
-    "serve:dev": "soul -d ./db/RCO2.sqlite -e $(realpath ./_devExtensions)",
+    "serve:dev": "node ./backend.js",
     "serve:large": "if [ ! -d dist ]; then echo '\\e[31mError: The \"dist\" folder does not exist. Please run \"yarn build\" first.\\e[0m'; else soul -d ./db/RCO2-large.sqlite -e $(realpath ./_extensions); fi",
```

Comment:
hello @TahaKhanAbdalli - I don't remember why we introduced `backend.js` - could you explain it to me please?  I see that we just use it for `serve:dev`.